processing/src/test/java/io/druid/query/topn/TopNQueryRunnerTest.java

```diff
@@ -3488,21 +3546,28 @@ public void testNumericDimensionTopNWithNullPreviousStop()
         .metric(new DimensionTopNMetricSpec(null, StringComparators.NUMERIC))
         .threshold(2)
         .intervals(QueryRunnerTestHelper.secondOnly)
-        .aggregators(Lists.<AggregatorFactory>newArrayList(QueryRunnerTestHelper.rowsCount))
+        .aggregators(duplicateAggregators(
+            QueryRunnerTestHelper.rowsCount,
+            new CountAggregatorFactory("rows1", "rows")
```

Comment:
I think simply changing its name from ```rows``` to ```rows1``` should work. The changes in ```CountAggregatorFactory``` are not necessary.